apps/web/src/components/HeroUpdated/Hero.tsx

```diff
@@ -35,32 +37,39 @@ const paddingMap = new Map([
   ],
 ]);
 
+interface NavTab {
```

Comment:
I'm not sure if we need to support different colours here, too.  Might be worth checking with Josh.

Reply:
Not even sure the NavTabs support different colours, was playing around with the primitive and couldn't get different colours to appear 😕

Reply:
Fair enough.  Please update the stories so that one of the tabs is "active" though.

Reply:
Sure, I have done so in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11713/commits/c4bbcbb7496f3d12506d481f50a045ae59ccfa0c